app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def sortable(column, title = nil)
+    title ||= column.titleize
+    css_class = column == sort_column ? "current #{sort_direction}" : nil
+    direction = column == sort_column && sort_direction == "asc" ? "desc" : "asc"
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.